Add more archive and audio file types to categories

tsweet64 opened the pull request:
These are some common (and some less common) archive and audio formats that I suggest adding to the defaults.

Opus, zst, and xz are very common. fsa (generated by fsarchiver) and tar.lzo are much less common, but still valid archives.

shundhammer commented:
There are literally thousands of filename extensions; I can't keep track of them all, only of some very common ones. That's one reason why this is configurable (Menu _Edit_ -> _Configure QDirStat_ -> Tab _MIME Categories_) so all users can add the ones they prefer to be highlighted (and change the categories and the colors, too).

Having said that, for more common ones I am open for pull requests so they are available for all first-time users, too; it's very easy:

https://github.com/shundhammer/qdirstat/blob/master/src/MimeCategorizer.cpp#L266-L447

See also

- https://github.com/shundhammer/qdirstat/blob/master/doc/Contributing.md
- https://github.com/shundhammer/qdirstat/blob/master/doc/GitHub-Workflow.md

tsweet64 commented:
> Having said that, for more common ones I am open for pull requests so they are available for all first-time users, too

With the exception of `fsa`, all of the archive formats I added in this pull request are the standard ones supported in GNU tar. In my view it makes sense for these ones to be available by default to users. I also added opus, a very common audio format, and ape, an older but still moderately common audio format.

Is there something in particular I should fix/add/remove here? I'm slightly confused. I certainly didn't expect you to keep track of every format in existence, which is why I attempted to make a PR adding some of the more common ones I came across so that new users would have them by default.

shundhammer commented:
Duh; I am stupid, I didn't look at the category here, coming here just from the GitHub notifications inbox. I didn't see it already _is_ a pull request.

shundhammer commented:
Merged.

Sorry for the confusion + thanks for your contribution!

tsweet64 commented:
No worries, I assumed that was what happened. Thanks for maintaining this project!
